client/landing/stepper/declarative-flow/internals/index.tsx

```diff
@@ -82,6 +87,7 @@ export const FlowRenderer: React.FC< { flow: Flow } > = ( { flow } ) => {
 				return (
 					<Route key={ path } path={ `/${ path }` }>
 						<div className={ classnames( flow.name, flow.classnames, kebabCase( path ) ) }>
+							<ProgressBar value={ progressValue * 100 } total={ 100 } color="#1e81cc" />
```

Comment:
Can we change the color to #117AC9 to be consistent with the rest of the UI? Also, is it possible to move this into SCSS?

Reply:
Good catch!